src/main/java/oss/fosslight/common/Url.java

```diff
@@ -69,6 +69,7 @@ public static final class LICENSE {
 		
 		public static final String VALIDATION = PATH + "/validation";
 		public static final String SAVE_AJAX = PATH + "/saveAjax";
+		public static final String SAVE_AJAX_CHECK_ADMIN = PATH + "_register";
```

Comment:
Is there a reason it's `_` and not `/`?

Reply:
It's my mistake. I chaged it!

Reply:
if you change code, please click `Resolve Conversation` button. Thank you 👍